src/docking/search_nci.f90

```diff
@@ -1116,15 +1125,30 @@ subroutine restart_xTB(env, mol, chk, calc, basisset)
       calc%maxiter = set%maxscciter
 
       call chk%wfn%allocate(mol%n, calc%basis%nshell, calc%basis%nao)
+      ! Make sure number of electrons is initialized an multiplicity is consistent
+      chk%wfn%nel = nint(sum(mol%z) - mol%chrg)
+      chk%wfn%nopen = mol%uhf
+      if (chk%wfn%nopen == 0 .and. mod(chk%wfn%nel, 2) /= 0) chk%wfn%nopen = 1
```

Comment:
What about molecules, for which `mol%uhf` has already been assigned to, e.g. `2`? Wouldn't this be (incorrectly) determined to `0`?

Reply:
Should not be the case. `chk%wfn%nopen` is set once to `mol%uhf` in line 1130 and only changed in line 1131 if `mol%uhf` is zero and the number of electrons is odd. This is similar to the `xtb` main routine:
https://github.com/grimme-lab/xtb/blob/043d587270844d953b0f5c1fe88e34a0eccb3b58/src/prog/main.F90#L600-L603